source/common/network/udp_listener_impl.cc

```diff
@@ -94,41 +68,60 @@ void UdpListenerImpl::onSocketEvent(short flags) {
 
 void UdpListenerImpl::handleReadCallback() {
   ENVOY_UDP_LOG(trace, "handleReadCallback");
-  sockaddr_storage addr;
-  socklen_t addr_len = 0;
-
+  // Max UDP payload.
+  constexpr uint64_t const read_buffer_length = 1500;
```

Comment:
Can we make this a constant? Presumably in the future we might want to support jumbo frames and have that be configurable somehow? Should we TODO/comment that?

Reply:
On the internet it is 1500, but within the datacenter it can be 9000, and using jumbo frames may be required if we want to do things like add a prefix to frames that we forward around during proxy (e.g., something like proxy proto for UDP/QUIC). I'm mainly suggesting that we may not always be doing 1500 with UDP code and we should plan on having this be flexible.